editor/blocks/quote/style.scss

```diff
@@ -0,0 +1,24 @@
+#editor blockquote {
```

Comment:
I think it's preferable to add a class to the `quote` block (Similar to this https://github.com/WordPress/gutenberg/blob/master/docs/tinymce-per-block/src/blocks/quote-block/form.js#L95). Other blocks may use the `blockquote` with other styling options.